src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -341,7 +341,7 @@ void ControlCompOutput(EnergyPlusData &state,
                 ZoneInterHalf.MaxResult = 1.0;
                 ZoneInterHalf.MinResult = 0.0;
                 // MaxResult is greater than MinResult so simulation control algorithm may proceed normally
-            } else if (ZoneInterHalf.MaxResult > ZoneInterHalf.MinResult) {
+            } else {
```

Comment:
[src/EnergyPlus/GeneralRoutines.cc:344]:(style),[knownConditionTrueFalse],Condition 'ZoneInterHalf.MaxResult>ZoneInterHalf.MinResult' is always true